packages/forms/resources/views/components/key-value.blade.php

```diff
@@ -86,7 +86,7 @@ class="text-danger-600 hover:text-danger-700"
                 </tbody>
             </table>
 
-            @if ($canAddRows() && $canEditKeys() && (! $isDisabled()))
+            @if ($canAddRows() && $canEditRows() && (! $isDisabled()))
```

Comment:
I've changed this, since the user can disables only keys or values editing.

If the user disables row editing, then it cannot add new rows, since this rows will cannot be edited.
If the user disables only key or value editing, then they can add new rows, because they can edit them separately